Documentation/gettingstarted/requirements-eks.rst

```diff
@@ -16,10 +16,35 @@ For more information on AWS ENI mode, see :ref:`ipam_eni`.
    If you want to chain Cilium on top of the AWS CNI, refer to the guide
    :ref:`chaining_aws_cni`.
 
-**Requirements:**
-
-* It is recommended to create an EKS cluster without any nodes, install
-  Cilium, and then scale up the number of nodes with Cilium already deployed.
+The following command creates a Kubernetes cluster with ``eksctl``
```

Comment:
I think we should also update the other `requirements-*.rst` files accordingly.

Reply:
I'm not sure how I missed this. Fixed.